llama-index-integrations/vector_stores/llama-index-vector-stores-lancedb/llama_index/vector_stores/lancedb/base.py

```diff
@@ -348,7 +348,7 @@ def add(
                     self._table_name, data, mode=self.mode
                 )
 
-            self._table.add(data, mode=self.mode)
+            self._table.add(data)
```

Comment:
perhaps we can just add **add_kwargs here to handle the designated mode parameter.

Reply:
@nerdai good point. I hadn't seen that there.  I updated the PR with that change.